app/models/container_volume.rb

```diff
@@ -1,4 +1,5 @@
 class ContainerVolume < ApplicationRecord
   include ReportableMixin
   belongs_to :parent, :polymorphic => true
+  belongs_to :persistent_volume_claim, :dependent => :destroy
```

Comment:
We should drop this. If it's a `ContainerVolume` then it has a direct polymorphic relationship with its parent (Pod).